Keas.Core/Models/PersonCategories.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Keas.Core.Models
+{
+    public class PersonCategories
+    {
+        const string Faculty = "Faculty";
+        const string Staff = "Staff";
+        const string AdminStaff = "Admin Staff";
+        const string GradStudent = "Grad Student";
+        const string Undergrad = "Undergrad";
+        const string Visitor = "Visitor";
+        const string Volunteer = "Volunteer";
+
+        public static List<string> Types = new List<string>
+        {
+            Faculty,
+            Staff,
+            AdminStaff,
+            GradStudent,
+            Undergrad,
+            Visitor,
+            Volunteer,
+        };
+
+        public static List<string> BulkEditChoices = new List<string>
+        {
+            "-- Not Set --",

```

Comment:
This is always one of those things where I'd rather leave this out (since it's not a model layer concern) and then have the controller create the SelectList with the bulkEditChoices plus a default choice and pass that to the view.  Would also make it easier to later compare without having to retype `"-- Not Set --"` and then any chance to those default value will need to be changed in 2 places.